tutorials/dev/use_pass_instrument.py

```diff
@@ -0,0 +1,206 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=line-too-long
+"""
+.. _tutorial-use-pass-instrument:
+
+How to Use TVM Pass Instrument
+==============================
+**Author**: `Chi-Wei Wang <https://github.com/chiwwang>`_
+
+As more and more passes are implemented, it becomes interesting to instrument
+passes execution, analyze per-pass effects and observe various events.
+We have extended :py:class:`tvm.transform.PassContext` to accept a list of
+instrument classes. Also a decorator :py:func:`tvm.ir.instrument.pass_instrument` is provided to easily implement instrument classes.
```

Comment:
Pass instrument framework extended ... instrument instances, Also a decorator :py:func:`tvm.instrument.pass_instrument` ... instrument instances.

Reply:
I think pass_instrument is used to help implement a "class", not a instance.
And argument "instruments" accept a list of instances. Do I understand correctly?

Reply:
yes, you are right.

So it should be,
Pass instrument framework extended ... instrument **instances**, Also a decorator :py:func:tvm.instrument.pass_instrument ... instrument **classes**.
Sorry for miss-copied the second one.

Reply:
I don't quite get what  "..." should be in "Pass instrument framework extended ... instrument instances".
But wordings around here is changed. Could you take a look again?